src/ontology/uberon-edit.obo

```diff
@@ -224667,6 +224685,71 @@ is_a: UBERON:0002553 ! anatomical cavity
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-07-05T11:30:55Z xsd:dateTime
 
+[Term]
+id: UBERON:8480023
+name: skin of lateral lumbar region of abdomen
+def: "A zone of skin that is part of the lateral lumbar region of abdomen." [https://orcid.org/0000-0002-0819-0473]
+synonym: "skin of flank" EXACT []
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0000180 ! lateral lumbar region of abdomen
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-08-29T14:11:43Z xsd:dateTime
+
+[Term]
+id: UBERON:8480024
+name: skin of sacral region
+def: "A zone of skin that is part of the sacral region." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0005473 ! sacral region
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-08-29T14:14:27Z xsd:dateTime
+
+[Term]
+id: UBERON:8480025
+name: skin of clavicle region
+def: "A zone of skin that is above the clavicle zone in the pectoral girdle region." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0001421 ! pectoral girdle region
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-08-29T14:17:12Z xsd:dateTime
+
+[Term]
+id: UBERON:8480026
+name: skin of iliac crest region
+def: "A zone of skin that is above the iliac crest in the pelvic girdle bone/zone." [https://orcid.org/0000-0002-0819-0473]
+synonym: "skin of posterior iliac crest region" BROAD []
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0007830 ! pelvic girdle bone/zone
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-08-29T14:19:16Z xsd:dateTime
+
+[Term]
+id: UBERON:8480027
+name: temple
```

Comment:
@bvarner-ebi - could you help a bit with this. I saw that there is temporalis muscle in uberon (http://purl.obolibrary.org/obo/UBERON_0001598) and this should have some relation to temple (I assume), but not 100% sure how it actually relates - would you mind adding the appropriate relation please? thanks :)

Reply:
@shawntanzk, I would be inclined to model the muscle wrt to bones. I would not add a relation between the muscle and a body region unless you see examples of this in Uberon.

Reply:
perhaps then adding bone relation to temple? Just thought ideally that muscle should have a relation to temple given the name, but if you think otherwise that's fine too :)

Reply:
> perhaps then adding bone relation to temple?

This sounds reasonable to me since the temple exists through the presence of the fused bones, not the presence of the muscle.

Reply:
You could add gloss regarding the muscle if you're keen on mentioning it.